tests/Microplate/Scalars/Column6Test.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Tests\Microplate\Scalars;
+
+use GraphQL\Error\Error;
+use GraphQL\Type\Definition\ScalarType;
+use MLL\Utils\Microplate\Scalars\Column6;
+use PHPUnit\Framework\TestCase;
+
+final class Column6Test extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (! class_exists(ScalarType::class)) {
+            self::markTestSkipped('Our webonyx/graphql-php version requires PHP 8');
+        }
+    }
+
+    public function testSerializeThrowsIfNotAnInt(): void
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('Value not in range 1-6: "2".');
+
+        (new Column6())->serialize('2');
+    }
+
+    public function testSerializeThrowsIfInvalid(): void
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('Value not in range 1-6: 7.');
+
+        (new Column6())->serialize(7);
+    }
+
+    public function testSerializePassesWhenValid(): void
+    {
+        self::assertSame(
+            2,
+            (new Column6())->serialize(2)
+        );
+    }
+
+    public function testParseValueThrowsIfInvalid(): void
+    {
+        $this->expectException(Error::class);
+        $this->expectExceptionMessage('Value not in range 1-6: 7.');
+
+        (new Column6())->parseValue(7);
```

Comment:
One more test please for lower bound of the int-range functionality. e.g. `(new Column6())->parseValue(0);` or `(new Column6())->parseValue(-1);`.

Reply:
I decided to remove all scalar tests in favor of the tests in `graphql-php-scalars`, see https://github.com/mll-lab/graphql-php-scalars/commit/080b5a2fdf368562511b7aceaa20e8738b7f5375.